index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio Website</title>
+</head>
+<body>
+  <header id="container">
+    <div class="head">
+        <h1 class="logo">Suleiman</h1>
```

Comment:
- [ ] For semantic reasons, kindly add a link place holder to your logo. 
eg
`<a href="url">link text</a>`